src/rmodels.c

```diff
@@ -4681,6 +4687,29 @@ static Image LoadImageFromCgltfImage(cgltf_image *cgltfImage, const char *texPat
     return image;
 }
 
+static BoneInfo *LoadGLTFBoneInfo(cgltf_skin skin, int *boneCount) {
```

Comment:
Please, could you review the parentheses to be aligned, following raylib coding conventions?